skin-resque-frontend/src/pages/CosmeticsCatalogue/CosmeticsCatalogue.scss

```diff
@@ -0,0 +1,122 @@
+.cosmetics-page {
+	z-index: 1;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+
+	.pink {
+		width: 100%;
+		height: 400px;
+		background-color: #F3B4C5;
+	}
+
+	.yellow-spot {
+		position: absolute;
+		width: 30%;
+		height: auto;
+		left: -40px;
+		top: 100px;
+	}
+	
+	.small-pink-dots {
+		position: absolute;
+		width: 50%;
+		height: auto;
+		left: 370px;
+		top: -17px;
+	}
+	
+	.pink-spot {
+		position: absolute;
+		width: 30%;
+		height: auto;
+		left: 1063px;
+		top: 100px;
+	}
+
+	.white {
+		z-index: 0;
+		background-color: white;
+		height: 800px;
+		width: 100%;
+
+		.rosemary-sprig-left {
+			position: absolute;
+			width: auto;
+			height: 40%;
+			left: -90px;
+			top: 655px;
+		}
+		
+		.rosemary-sprig-right {
+			position: absolute;
+			width: auto;
+			height: 40%;
+			left: 1110px;
+			top: 1010px;
+		}
+	}	
+
+	.catalogue {
+		position: absolute;
+		top: 200px;
+		background-color: #F6EFF0;
+		width: 80%;
+		height: auto;
+		display: flex;
+		flex-direction: column;
+		gap: 20px;
+		align-items: center;
+
+		.page-title {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			padding: 20px;
+		}
+
+		.cosmetics-list {
+			justify-content: center;
+			align-items: center;
+			display: flex;
+			gap: 100px;
+			flex-wrap: wrap;
+			width: 80%;
+
+			.cosmetic {
+				display: flex;
+				flex-direction: column;
+				justify-content: space-between;
+				align-items: center;
+				background-color: #EEE1E3;
+				height: 300px;
+				width: 250px;
+
+				&:hover {
+					filter: drop-shadow(30px 10px 4px #D3D3D3);
```

Comment:
The drop-shadow values are too rough for me, mabye we should change that to the values from figma?
`box-shadow: 66px 96px 47px rgba(49, 28, 31, 0.01), 37px 54px 39px rgba(49, 28, 31, 0.05), 17px 24px 29px rgba(49, 28, 31, 0.09), 4px 6px 16px rgba(49, 28, 31, 0.1), 0px 0px 0px rgba(49, 28, 31, 0.1);`

Reply:
I changed it and it looks much better now